FlowCrypt/Functionality/Services/Organisational Rules Service/ClientConfigurationProvider.swift

```diff
@@ -36,6 +34,10 @@ extension ClientConfigurationProvider: ClientConfigurationProviderType {
     }
 
     func save(clientConfiguration: ClientConfiguration) {
-        clientConfigurationCache.save(ClientConfigurationObject(clientConfiguration, user: EncryptedStorage().activeUser))
+        guard let user = clientConfigurationCache.encryptedStorage.activeUser else {
+            assertionFailure("Internal inconsistency. Missed client configuration")
```

Comment:
@tomholub Or maybe better to rework to show this as error for the user?

Reply:
I think this would be a fatal error. Anyway that will be covered in #387